<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveVariables</class>
 <widget class="QWidget" name="SaveVariables">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>E&amp;dit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tblVars">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>SaveVariables</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>SaveVariables</receiver>
   <slot>removeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblVars</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SaveVariables</receiver>
   <slot>tableClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdit</sender>
   <signal>clicked()</signal>
   <receiver>SaveVariables</receiver>
   <slot>editCurrentVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblVars</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>SaveVariables</receiver>
   <slot>editCurrentVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addClicked()</slot>
  <slot>removeClicked()</slot>
  <slot>tableClicked(QModelIndex)</slot>
  <slot>editCurrentVar()</slot>
 </slots>
</ui>
